engine/installation/index.md

```diff
@@ -8,51 +8,30 @@ redirect_from:
 title: Install Docker
 ---
 
-## Docker variants
+Docker is available in two editions: **Community Edition (CE)** and **Enterprise
+Edition (EE)**.
 
-Docker is available in multiple variants:
+Docker Community Edition (CE) is ideal for developers and small
+teams looking to get started with Docker and experimenting with container-based
+apps. For more information about Docker CE, see
+[Docker Community Edition](https://www.docker.com/community-edition/){: target="_blank" class="_" }.
 
-- **Docker Enterprise Edition (Docker EE)** is designed for enterprise
-  development and IT teams who build, ship, and run business critical
-  applications in production at scale. Docker EE is integrated, certified, and
-  supported to provide enterprises with the most secure container platform in
-  the industry to modernize all applications. For more information
-  about Docker EE, including purchasing options, see
-  [Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
+Docker Enterprise Edition (EE) is designed for enterprise
+development and IT teams who build, ship, and run business critical
+applications in production at scale. For more information about Docker EE,
+including purchasing options, see
+[Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
 
-- **Docker Community Edition (Docker CE)** is ideal for developers and small
-  teams looking to get started with Docker and experimenting with
-  container-based apps. Docker CE is available on many platforms, from desktop
-  to cloud to server. Docker CE is available for macOS and Windows and provides
-  a native experience to help you focus on learning Docker. You can build and
-  share containers and automate the development pipeline all from a single
-  environment.
+{% include docker_ce_ee.md %}
 
-  Docker CE gives you the option to run **stable** or **edge** builds.
```

Comment:
> Removed mentions to stable and edge, because we're already talking about them once you click on the platform where you want to install Docker.

I can add it back, but right now we are presenting a ton of options upfront. You need to understand the difference between CE and EE, and if it exists for your platform.
Since edge is not for all users, and once you navigate to the platform we present that choice there, I think we can remove a bit of the complexity here.